src/features/search/Search.tsx

```diff
@@ -40,7 +40,7 @@ const Search: React.FC<{}> = () => {
   // if the query string ever changes, fetch new data
   useEffect(() => {
     dispatch(loadSearchResults(searchParams))
-  }, [q, page, sort, dispatch, searchParams])
+  }, [q, page, sort, dispatch])
```

Comment:
`searchParams` start a feedback loop and goes infinitely spamming search requests.